src/components/WindowControls/WindowControls.vue

```diff
@@ -11,7 +11,8 @@
 
 <script lang="ts">
```

Comment:
AFAIK the `WindowControls` was used to add window controls to the Windows build

But when we worked on the build, I set the Windows build window controls to be the native Windows controls

So, if this component is not necessary for the Mac build, we can delete it

Reply:
I think you can submit an issue to track it, not in this PR, right?